core/main/src/Options/Classes/Options.ts

```diff
@@ -77,6 +78,7 @@ export class Options implements IOptions, IOptionLoader<IOptions> {
         this.motion = new Motion();
         this.particles = new Particles();
         this.pauseOnBlur = true;
+        this.pauseOnOutsideViewport = true;
```

Comment:
New features should be disabled by default so an upgrading user won't find differences in his configuration